rfc/5/index.md

```diff
@@ -308,124 +374,38 @@ Conforming readers:
-may not correspond to the measurement at the same array coordinates in instrument 2
-(i.e., it may not be the same physical location in the sample).
-To analyze both images together, they must be transformed to a common coordinate system.
-
-The set of coordinate transformations encodes relationships between coordinate systems,
-specifically, how to convert points from one coordinate system to another.
-Implementations can apply the coordinate transform to images or points
-in coordinate system "sampleA_instrument2" to bring them into the "sampleA_instrument1" coordinate system.
-In this case, image data within the ROI defined in image2 should be transformed to the "sampleA_image1" coordinate system,
-then used for quantification with the instrument 1 image.
-
-The `coordinateTransformations` in the parent-level metadata would contain the following data.
-The transformation parameters are stored in a separate zarr-group
-under `coordinateTransformations/sampleA_instrument2-to-instrument1` as shown above.
-
-```json
-"coordinateTransformations": [
-    {
-        "type": "affine",
-        "path": "coordinateTransformations/sampleA_instrument2-to-instrument1",
-        "input": "sampleA_instrument2",
-        "output": "sampleA_instrument1"
-    }
-]
-```
-
-And the image at the path `sampleA_instrument1` would have the following as the first coordinate system:
-
-```json
-"coordinateSystems": [
-    {
-        "name": "sampleA-instrument1",
-        "axes": [
-            {"name": "z", "type": "space", "unit": "micrometer"},
-            {"name": "y", "type": "space", "unit": "micrometer"},
-            {"name": "x", "type": "space", "unit": "micrometer"}
-        ]
-    },
-]
-```
-
-The image at path `sampleA_instrument2` would have this as the first listed coordinate system:
-
-```json
-[
-    {
-        "name": "sampleA-instrument2",
-        "axes": [
-            {"name": "z", "type": "space", "unit": "micrometer"},
-            {"name": "y", "type": "space", "unit": "micrometer"},
-            {"name": "x", "type": "space", "unit": "micrometer"}
-        ]
-    }
-],
-```
-````
+- **Inside `multiscales > datasets`**: `coordinateTransformations` herein MUST be restricted
+  to a single `scale`, `identity` or `sequence` of translation and scale transformations.
+  Fore information, see [multiscales section below](#multiscales-metadata).
+- **Inside `multiscales > coordinateTransformations`: Additional transformations for single multiscale images MAY be stored here.
+  The `coordinateTransformations` field MUST contain a list of valid [transformations](#transformation-types).
+  The input to these transformations MUST be the intrinsic coordinate system.
+  The output can be another coordinate system defined under multiscales > coordinateSystems.
+  
+- **Inside `Scene > coordinateTransformations`**: Transformations between two or more images
+  MUST be stored in the attributes of a `Scene` in a parent zarr group.
```

Comment:
This is the first mention of `Scene` in this document. Reading below "'Scene' dictionary is located under the custom attributes of a parent-level zarr-group", it's not very clear whether this means:

```
{
  "ome": {
    "scene": {
      "coordinateTransformations": [],
      "coordinateSystems": []
    }
  }
}
```
or something else?

If this is correct then here it should be:
 **Inside `scene > coordinateTransformations`**: ...in the attributes of a `scene` dictionary in a parent zarr group.